float64 q1
float64 q2
float64 q3
---
float64[] ee_pos_or
